features/step_definitions/journey/registration_steps.rb

```diff
@@ -0,0 +1,70 @@
+# Group pieces together when they are always consecutive (renewal - new lower tier - new upper tier)
+# Helpers vs Step to group things together - Choose helpers, avoid steps.
+# Helpers vs Step - repeat helpers, not steps.
+# Use `Given` to set up instance variables for the rest of the journey decisions
+
+Given("I want to register as a lower tier carrier") do
+  load_all_apps
+
+  @tier = :lower
```

Comment:
Elsewhere in the pack I've specified the tier as a string, e.g. `"lower"` . I'm happy for it to be a symbol instead, but can we change all `@tier` variables if we're doing this?

Reply:
```suggestion
  @tier = "lower"
```